### PR TITLE
Configure Psycopg to be used with gevent in non-blocking way

### PR DESCRIPTION
**Pull request by snopoke**
@NoahCarnahan 

cc @czue @dannyroberts

### CODE REVIEW

**Comment by snopoke** [comment]
This should give our django workers a boost :rocket:

File: manage.py [hunk 1]
```diff
@@ -59,6 +59,7 @@ def _set_source_root(source_root):
     if len(sys.argv) > 1 and sys.argv[1] in GEVENT_COMMANDS:
         from restkit.session import set_session; set_session("gevent")
         from gevent.monkey import patch_all; patch_all(subprocess=True)
+        from psycogreen.gevent import patch_psycopg; patch_psycopg()
```
Comment (dimagimon) [inline manage.py]: E702 multiple statements on one line (semicolon)

**Comment by czue** [comment]
@snopoke doing a quick sanity check before we merge

**Comment by snopoke** [comment]
:white_check_mark:

**Comment by dannyroberts** [comment]
Woah! Does this mean that up until now, greenlets don't yield when they make requests to postgres? Implying that while a function that made a few calls to postgres (and no other i/o) was running for a few seconds, no other requests could be processed by that worker? Great discovery!

**Comment by snopoke** [comment]
Yup, exactly

**Pull request merged**